mds_array_manipulation/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "mds_array_manipulation"
+version = "0.1.0"
```

Comment:
Can set it to 0.0.1 to match the [release version for milestone 1?](https://pages.github.ubc.ca/MDS-2023-24/DSCI_524_collab-sw-dev_students/release/milestone1/milestone1.html#creating-a-release-on-github.com)